Tests/SwiftCurrent_SwiftUITests/SwiftCurrent_ModalTests.swift

```diff
@@ -134,16 +133,8 @@ final class SwiftCurrent_ModalTests: XCTestCase, Scene {
             var _workflowPointer: AnyFlowRepresentable?
             var body: some View { Text("FR7 type") }
         }
-        var model: WorkflowViewModel!
-        var launcher: Launcher!
-        var fr1: InspectableView<ViewType.View<WorkflowItem<FR1, WorkflowItem<FR2, WorkflowItem<FR3, WorkflowItem<FR4, WorkflowItem<FR5, WorkflowItem<FR6, WorkflowItem<FR7, Never, FR7>, FR6>, FR5>, FR4>, FR3>, FR2>, FR1>>>!
```

Comment:
😱

Reply:
Isn't it great to get rid of all that? My favorite is the final characters: `>>>!`

Reply:
Yeah very happy to see these go